rust/operator-binary/src/security/oidc.rs

```diff
@@ -25,15 +31,21 @@ pub enum Error {
     },
 
     #[snafu(display(
-        "found existing secret [{}/{}], but key {} is missing",
-        name,
-        namespace,
-        STACKABLE_OIDC_ADMIN_PASSWORD_KEY
+        "found existing admin password secret {secret:?}, but the key {STACKABLE_OIDC_ADMIN_PASSWORD_KEY} is missing",
     ))]
-    OidcAdminPasswordKeyMissing { name: String, namespace: String },
+    OidcAdminPasswordKeyMissing { secret: ObjectRef<Secret> },
+
+    #[snafu(display("invalid OIDC well known URL"))]
+    InvalidOidcWellKnownUrl {
+        source: stackable_operator::commons::authentication::oidc::Error,
+    },
+
+    #[snafu(display("Nifi doesn't support skipping the OIDC TLS verification"))]
+    NoOidcTlsVerificationNotSupported {},
```

Comment:
```suggestion
    UnsupportedTlsVerification {},
```

_Note: This also needs updating of the context selector below_

Reply:
I think the original name was more descriptive and would prefer to not lose the descriptiveness.
Maybe we can agree on something a bit more readable such as `SkippingOidcTlsVerificationNotSupported`, `SkippingOidcTlsVerificationUnsupported` or `UnsupportedOidcTlsVerificationSkipping` (although I prefer the first ^^)?

Reply:
I'm fine with `SkippingTlsVerificationNotSupported`.